src/app_service/service/message/async_tasks.nim

```diff
@@ -125,19 +125,16 @@ const asyncMarkAllMessagesReadTask: Task = proc(argEncoded: string) {.gcsafe, ni
 
   let response =  status_go.markAllMessagesFromChatWithIdAsRead(arg.chatId)
 
-  var activityCenterNotifications: JsonNode
+  var activityCenterNotifications: JsonNode = newJObject()
```

Comment:
Is it the case that we want to always pass the activityCenterNotifications object even when it's empty? why is it so? I thought that the field can be nil hence not added to the responseJson making the AC notification processing component less time. Can you tell me more why we are always forcing the AC notifications to be present please?

Reply:
In this particular case, the choice between a null value or a null object (pattern) is a matter of taste (). I chose the null object for reasons:
- this is the approach elsewhere in the project (a very bad reason :) )
- it seems safer to me, since the code below crashes:
```
var nullJson: JsonNode # missing `= newJObject()` here
    if nullJson.kind == JNull:
      ...
```
- and there is no need to do a check.